_lessons/python-scatter/00-introduction.md

```diff
@@ -46,38 +73,44 @@ The sum of the contributions from each segments gives the total scattered wave.
 
 Notice the small wave amplitude on the shadow side of the obstacle. 
 
-## How to run the scatter code
+### How to run the scatter code
 
 You can run the code interactively
 ```
 python scatter.py
 ```
 or by submitting a job to the scheduler. On Mahuika
 ```
-srun --account="myAccount" python scatter.py
+srun python scatter.py
 ```
-where "myAccount" is your account on Mahuika (e.g. nesi12345). This will launch a single task. 
+where you might have to pass `--account="myAccount"` as well as other options to the `srun` command. [See](https://nesi.github.io/hpc_training/lessons/maui-and-mahuika/slurm) for more information.
```

Comment:
@pletzer I think it would be better to link to the support.nesi documentation as I don't think the github training material is being updated any more?

Reply:
Maybe this one: https://support.nesi.org.nz/hc/en-gb/articles/360000359576-Slurm-Usage-A-Primer?

Reply:
Thanks @chrisdjscott. I changed the link.